cmd/erda-server/bootstrap.yaml

```diff
@@ -23,6 +23,9 @@ health:
 component-protocol:
 openapi-component-protocol:
 
+prometheus:
+  router_label_enable: false
```

Comment:
What's `router_label_enable`, how about set it to `true`.

Reply:
the `router_label_enable` default value is true, it find svc named `http-router@admin`

we should open the metrics in 9095 `http-server`, os set it to false